### PR TITLE
{data}[system/system] PostgreSQL v12.4

### PR DESCRIPTION
**Pull request by mboisson**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nuc.localdomain - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/0b76cb71c2b97b6a6d9a780405215e9d for a full test report.

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+configopts = '--with-python --with-openssl'
+
+sanity_check_paths = {
+    'files': ['bin/psql', 'bin/pg_config', 'lib/libpq.a', 'lib/libpq.%s' % SHLIB_EXT],
+    'dirs': [''],
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb:47]: @mboisson any reason why you removed the directory from the `sanity_check_paths`?

Comment (mboisson) [reply easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb:47]: Mmm, this is a new recipe, so I did not "remove" anything :P, but I'll see what could be missing

Comment (verdurin) [reply easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb]: Yes, I suppose it looks removed in the `review-pr` tool. Thanks.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/7c41ccef45e231792fa03ca791b66fb2 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @mboisson!

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb (hunk 1 above)
Comment (mboisson) [reply easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb]: mmmm, @verdurin, I don't have that directory in my installation. So adding this sanity_check makes a reinstallation fail...

**Pull request merged**